x-pack/plugins/apm/public/components/shared/KueryBar/index.tsx

```diff
@@ -5,7 +5,7 @@
  */
 
 import React, { useState, useEffect } from 'react';
-import { uniqueId, startsWith } from 'lodash';
+import { uniqueId, startsWith, find } from 'lodash';
```

Comment:
Yeah, I was using `find` because it supports object lookups, but probably not worth it. Your suggestion looks better anyway, so I'll just go with that :)